antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -6,10 +6,12 @@
   window.addEventListener('load', () => {
     const client = algoliasearch('BH4D9OD16A', '16e3a9155a136e4962dc4c206f8278bd')
     const index = client.initIndex('apache_camel')
-    const search = document.querySelector('#search')
+    const search = document.querySelector('.search')
     const container = search.parentNode
-    const results = document.querySelector('#search_results')
-    const cancel = document.querySelector('#search-cancel')
+    const results = document.querySelector('.search-results')
+    const cancel = document.querySelector('.search-cancel')
+    const className = 'navbar-search'
+    const hiddenClass = 'navbar-search results-hidden'
```

Comment:
Do we still need these changes to the JS code now that we have only one search input?

Reply:
No, I will change to use search cancel and search results by Id and revert the changes for this to normal.